settings.gradle

```diff
@@ -1 +1,15 @@
+pluginManagement.repositories {
+    mavenLocal()
+    maven {
+        url 'https://artifactory.rightmesh.io/artifactory/maven'
+        credentials {
+            username rightmesh_devportal_username
+            password rightmesh_devportal_password
```

Comment:
This references the right variable? This might need to be the artifactory password not the dev portal one.

Reply:
sadface emoji

Reply:
evensadderfaceemoji.svg